webpack.config.js

```diff
@@ -12,7 +12,6 @@ const isTest = env === 'test'
 
 const plugins = [
   new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify(env),
```

Comment:
are you sure about that?

Reply:
It could be in any other package, like in the "warning" package we are using

Reply:
please get this one back before merging

Reply:
Ohhh, yeah, removed that because we are using the babel-plugin-inline-env-variables now, but forgot that this could still be in node_modules.